cirq/study/flatten_expressions.py

```diff
@@ -337,7 +337,10 @@ def transform_sweep(self,
             sweep: The sweep to transform.
         """
         sweep = sweepable.to_sweep(sweep)
-        return _TransformedSweep(sweep, dict(self))
+        return sweeps.ListSweep([
+            dict(param_tuples) for param_tuples in _TransformedSweep(
```

Comment:
Delete the `_TransformedSweep` class and move the needed code from `_TransformedSweep.param_tuples` to here.

Reply:
Done.  Collapsed it into one function.